packages/storage-driver-local/src/index.ts

```diff
@@ -75,10 +75,20 @@ export class DriverLocal implements Driver {
 	}
 
 	async write(filepath: string, content: Readable) {
-		const fullPath = this.fullPath(filepath);
-		await this.ensureDir(dirname(fullPath));
-		const writeStream = createWriteStream(fullPath);
-		await pipeline(content, writeStream);
+		try {
+			const fullPath = this.fullPath(filepath);
+			await this.ensureDir(dirname(fullPath));
+			const writeStream = createWriteStream(fullPath);
+			content.on('close', () => {
+				writeStream.end();
+				writeStream.on('finish', () => {
+					writeStream.close();
+				});
+			});
```

Comment:
This has actually worsened things in my tests, but I assume that was only added for testing purposes and will be removed again

Reply:
in testing with myself and @br41nslug we found having that listener had the opposite effect, but will test it with your changes above and see.

Reply:
Great, thanks! 👍